PySDM/physics/saturation_vapour_pressure/bolton_1980.py

```diff
@@ -0,0 +1,28 @@
+"""
+[Bolton 1980](https://doi.org/10.1175/1520-0493(1980)108<1046:TCOEPT>2.0.CO;2)
+"""
+
+import numpy as np
+
+
+class Bolton1980:
+    def __init__(self, _):
+        pass
+
+    @staticmethod
+    def pvs_Celsius(const, T):
```

Comment:
note that the function name indicates that T is in Celsius, the coefficients given in the paper are for T in Kelvins

Reply:
within PySDM codebase, the saturation vapour pressures formulae are used through the `pvs_Celsius` API (worth changing at some point to Kelvins, BTW), so barely changing the function name here will render it unusable with all other parts of the code. Instead, we need to convert Celsius to Kelvin within the function for now